components/tidb_query/src/expr/builtin_string.rs

```diff
@@ -914,6 +914,37 @@ impl ScalarFunc {
             .map(|i| 1 + i as i64)
             .or(Some(0)))
     }
+
+    #[inline]
+    // See https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_quote
```

Comment:
Prefer to put comments above of attributes. Actually I think we don't need to give an inline hint here.

Reply:
will remove it.